.github/workflows/apply-knowledge.yaml

```diff
@@ -15,6 +15,11 @@ name: Apply Devin Knowledge
 on:
   workflow_dispatch:
 
+env:
```

Comment:
For each of these actions, now that we have this env defined at the workflow level, we can remove the envs defined at the job level